package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@schibstedspain/openads-appnexus-prebid",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
the configuration specification is incompatible with the 1.2.0 configuration, so it should be a major version update 2.0.0

Reply:
you are right